_posts/2021-10-24-g2p-advanced-mappings.md

```diff
@@ -22,6 +22,10 @@ This is the 4th blog post in a seven-part series about g2p. This is a relatively
 - [Preprocessing mappings]({{ "g2p-preprocess" | absolute_url }})
 - [Contributing]({{ "g2p-contributing" | absolute_url }})
 
+## NOTE!
+
+As of September 2023, there is a new version of `g2p` available: 2.0 - the instructions in this blog were originally written for version 1.x. If you already have `g2p` installed, we recommend that you upgrade your installation before continuing on with this post.
```

Comment:
Ditto. Is it OK to leave this as is, or should be change it to March, 2024?

Reply:
I guess we did the 2.0 breaking changes back in september, but it might still be confusing to have pypi and this disagree on the 2.0 date.